src/styles/blocks/container.scss

```
@@ -0,0 +1,22 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  width: 1200px;
+  padding: 50px 128px;
```

```suggestion
  width: <calculate value here>;
  padding-top: 50px;
  margin: auto; 
```

It can help with test: 
1. Do not hardcode width, it should base on the card width and gap between cards (should be about 944px)
2. Centering div using` margin:auto;`  (not it looks like not centered on page), and after that you don't need to set paddings